packages/ui/src/ui/pages/navigation/modals/AttributesEditor.tsx

```diff
@@ -44,11 +45,37 @@ import {docsUrl} from '../../../config';
 
 import './AttributesEditor.scss';
 import UIFactory from '../../../UIFactory';
-import {FIX_MY_TYPE} from '../../../../@types/types';
 
 const block = cn('navigation-attributes-editor');
 
-export function AttributesEditorLoaded() {
+interface FormValues {
+    general: {
+        path: Array<{title: string}>;
+        account: string;
+        primary_medium: string;
+        tablet_cell_bundle: string;
+        in_memory_mode: InMemoryModeType | '';
+    };
+    storage: {
+        path: Array<{title: string}>;
+        optimize_for: 'lookup';
+        compression_codec: Array<string>;
+        erasure_codec: string;
+        replication_factor: number;
+
+        expiration_time: {from?: string | number};
```

Comment:
I think it is better to use `{to: string | number}` DatePicker prop for `expiration_time` value.

> In meaning: date when smth will be deleted

Reply:
It is ok to use `from` prop here, but Datepicker also have `to` prop which works same - just looks more suitable to describe `expiration_time`

For me it is not an issue, just my improvement thoughts 👌🏻

Reply:
`to` is defined only for ranges:
<img width="552" alt="image" src="https://github.com/ytsaurus/ytsaurus-ui/assets/64027148/db2d54ac-7423-4fc6-806e-b977a1230209">